community/contributing/nightly-builds/index.md

```diff
@@ -150,3 +149,28 @@ All processes are run as the `jenkins` user and most of the useful links are in
 ```bash
 $ sudo su - jenkins
 ```
+
+### Common Build Failures
```

Comment:
There's already an h3-level `Build failures` heading above Should this go under there?

Reply:
Yep, thanks.